templates/page.gzg.json

```diff
@@ -0,0 +1,118 @@
+    },
+    "list": {
+      "type": "list",
+      "settings": {
+        "color_scheme": "background-1",
+        "width": "narrow",
+        "list_type": "ordered"
+      },
+      "blocks": {
+        "bullet1": {
+          "type": "list_item",
+          "settings": {
+            "text": "<p>Geben Sie Ihre Bestellnummer und Postleitzahl im Formular unten ein.</p>"
+          }
+        },
+        "bullet2": {
+          "type": "list_item",
+          "settings": {
+            "text": "<p>Wählen Sie aus, welche Produkte aus Ihrer Bestellung Sie zurücksenden möchten.</p>"
+          }
+        },
+        "bullet3": {
+          "type": "list_item",
+          "settings": {
+            "text": "<p>Drucken Sie das Rücksendedokument aus und legen Sie es Ihrer Rücksendung bei.</p>"
+          }
+        }
+      },
+      "block_order": ["bullet1", "bullet2", "bullet3"]
+    },
+    "rich2": {
+      "type": "rich-text",
+      "settings": {
+        "color_scheme": "background-1",
+        "width": "narrow",
+        "marginAbove": false,
+        "marginBelow": false
+      },
+      "blocks": {
+        "text": {
+          "type": "text",
+          "settings": {
+            "hyphenation": "de",
+            "text_align": "ta-l",
+            "text": "<p>Sobald die Rücksendung bei uns eingegangen ist, erstatten wir Ihnen den vollen Betrag der Bestellung zurück. Dafür verwenden wir die Zahlungsmethode, mit der Sie die Bestellung bei uns getätigt haben.<br><br><strong>Rücksendekosten können wir leider nicht erstatten.</strong> Die Erstattung erfolgt in der Regel innerhalb von fünf Tagen nach Eingang der Rücksendung.<br><br>Die Adresse für die Rücksendung ist:</p>"
+          }
+        }
+      },
+      "block_order": ["text"]
+    },
+    "address": {
+      "type": "address",
+      "settings": {
+        "color_scheme": "background-1",
+        "width": "narrow",
+        "text": "<p>sleep.ink Versandzentrum c/o SKF e.V.<br>Pankstraße 8-10<br>Haus D<br>13127 Berlin</p>"
+      }
+    },
+    "rich3": {
+      "type": "rich-text",
+      "settings": {
+        "color_scheme": "background-1",
+        "width": "narrow",
+        "marginAbove": false,
+        "marginBelow": true
+      },
+      "blocks": {
+        "text": {
+          "type": "text",
+          "settings": {
+            "hyphenation": "de",
+            "text_align": "ta-l",
+            "text": "<p><strong>Bitte beachten Sie, dass Rücksendungen ohne beigefügtes Rücksendedokument nicht verarbeitet werden können.</strong><br><br>Sollten Sie Fragen zur Geld-zurück-Garantie haben, melden Sie sich gerne bei unserem Kundenservice unter <a href='mailto:hallo@sleep.ink'>hallo@sleep.ink</a> oder nutzen Sie das Chatfenster auf der Webseite.</p>"
+          }
+        }
+      },
+      "block_order": ["text"]
+    },
+    "iframe": {
+      "type": "iframe",
+      "settings": {
+        "title": "",
+        "width": "wide",
+        "url": "https://sleep.ink/archive/gzg"
```

Comment:
Ein bisschen Out-of-Scope, aber:

Wenn wir die Domain wechseln bricht der Link. Wir sollten eine extra sub-Domain anlegen für die React-Componenten, die wir per iFrame einbinden. Sowas wie forms.sleep.ink oder so.

Reply:
Wenn sich die URL ändert, kann man das immer noch im Theme-Editor ändern. Wir werden mit der gesamten Seite die Domain wechseln und die Seiten, die wir dann nicht mehr brauchen, unpublishen.